dev/src/util.ts

```diff
@@ -14,6 +14,24 @@
  * limitations under the License.
  */
 
+// Untyped Number alias we can use to check for ES6 methods / properties.
+// tslint:disable-next-line:no-any variable-name
+const NumberAsAny = Number as any;
+
+/**
+ * Minimum safe integer in Javascript because of floating point precision.
+ * Added to not rely on ES6 features.
+ */
+export let MIN_SAFE_INTEGER: number =
+  NumberAsAny.MIN_SAFE_INTEGER || -(Math.pow(2, 53) - 1);
```

Comment:
Feel free to ignore since I think you can just remove the changes in this file: You shouldn't have to use `NumberAsAny`. I am surprised `Number. MIN_SAFE_INTEGER` doesn't work for you.